compiler/dialects/kernel.py

```diff
@@ -30,9 +30,12 @@ class BinaryOp:
     result = result_def(IntegerType)
 
 
-class QuantizedBinaryOp(BinaryOp):
+class QuantizedBinaryOp:
+    lhs = operand_def(IntegerType)
+    rhs = operand_def(IntegerType)
     zp_lhs = operand_def(IntegerType)
     zp_rhs = operand_def(IntegerType)
+    result = result_def(IntegerType)
 
```

Comment:
this was necessary because otherwise the ordering of operands is wrong, if i used inheritance the ordering was (zp_lhs, zp_rhs, lhs, rhs) and there was seemingly no way to change this